packages/hub-filecoin/package.json

```diff
@@ -39,24 +29,5 @@
     "@textile/security": "^0.6.2",
     "event-iterator": "^2.0.0",
     "loglevel": "^1.6.8"
-  },
-  "devDependencies": {
```

Comment:
bunch copy-pasta deps that aren't needed here. also references are now auto-added to tsconfig